chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -6,8 +6,14 @@
   "Microsoft-Hyper-V" => {
     "restart" => false
   },
-  "Microsoft-Hyper-V-Management-PowerShell" => {
-    "restart" => true
+  "RSAT-Hyper-V-Tools-Feature" => {
+    "restart" => false,
+  },
+  "Microsoft-Hyper-V-Management-Clients" => {
+    "restart" => false,
+  },
+  "Microsoft-Hyper-V-Management-Powershell" => {
```

Comment:
displayed from dism is Powershell, but windows is case insensitive anyway. So this works as is.